tests/v2/validation/rbac/rbac_test.go

```diff
@@ -0,0 +1,249 @@
+	standardUser       *management.User
+	standardUserClient *rancher.Client
+	session            *session.Session
+	cluster            *management.Cluster
+	adminProject       *management.Project
+}
+
+func (rb *rbTestSuite) TearDownSuite() {
+	rb.session.Cleanup()
+}
+
+func (rb *rbTestSuite) SetupSuite() {
+	testSession := session.NewSession(rb.T())
+	rb.session = testSession
+
+	client, err := rancher.NewClient("", testSession)
+	require.NoError(rb.T(), err)
+
+	rb.client = client
+
+	//Get cluster name from the config file and append cluster details in rb
+	clusterName := client.RancherConfig.ClusterName
+	require.NotEmptyf(rb.T(), clusterName, "Cluster name to install should be set")
+	clusterID, err := clusters.GetClusterIDByName(rb.client, clusterName)
+	require.NoError(rb.T(), err, "Error getting cluster ID")
+	rb.cluster, err = rb.client.Management.Cluster.ByID(clusterID)
+	require.NoError(rb.T(), err)
+
+}
+
+func (rb *rbTestSuite) ValidateListCluster(role string) {
+
+	//Testcase1 Verify cluster members - Owner/member are able to list clusters
+	clusterList, err := listClusters(rb.standardUserClient)
+	require.NoError(rb.T(), err)
+	assert.Equal(rb.T(), 1, len(clusterList.Data))
+	assert.Equal(rb.T(), rb.cluster.ID, clusterList.Data[0].Status.ClusterName)
+}
+
+func (rb *rbTestSuite) ValidateListProjects(role string) {
+
+	//Testcase2 Verify members of cluster are able to list the projects in a cluster
+	//Get project list as an admin
+	projectlistAdmin, err := listProjects(rb.client, rb.cluster.ID)
+	require.NoError(rb.T(), err)
+	//Get project list as a cluster owner/member
+	projectlistClusterMembers, err := listProjects(rb.standardUserClient, rb.cluster.ID)
+	require.NoError(rb.T(), err)
+	switch role {
+	case roleOwner:
+		//assert length of projects list obtained as an admin and a cluster owner are equal
+		assert.Equal(rb.T(), len(projectlistAdmin), len(projectlistClusterMembers))
+		//assert projects values obtained as an admin and the cluster owner are the same
+		assert.Equal(rb.T(), projectlistAdmin, projectlistClusterMembers)
+	case roleMember:
+		//assert projects list obtained as a cluster member is empty
+		assert.Equal(rb.T(), 0, len(projectlistClusterMembers))
+	}
+}
+
+func (rb *rbTestSuite) ValidateCreateProjects(role string) {
+
+	//Testcase3 Validate if cluster members can create a project in the downstream cluster
+	createProjectAsClusterMembers, err := createProject(rb.client, rb.cluster.ID)
+	require.NoError(rb.T(), err)
+	log.Info("Created project as a ", role, " is ", createProjectAsClusterMembers.Name)
+	require.NoError(rb.T(), err)
+	actualStatus := fmt.Sprintf("%v", rb.adminProject.State)
+	assert.Equal(rb.T(), "active", actualStatus)
+
+}
+
+func (rb *rbTestSuite) ValidateNS(role string) {
+
+	//Testcase4 Validate if cluster members can create namespaces in project they are not owner of
+	log.Info("Testcase4 - Validating if ", role, " can create namespace in a project they are not owner of. ")
```

Comment:
I agree with Ugur, having these log messages in conjunction with the name of the test is a little redunant

Reply:
I have removed the log messages for validations that have a test name. but for this test, since its a more generic case `ValidateNS` which includes 3 test cases, I have kept the log messages for these 3 test cases.